client/components/plans/plan-list/index.jsx

```diff
@@ -78,6 +79,10 @@ const PlanList = React.createClass( {
 			const filteredPlans = filterPlansBySiteAndProps( plans, site, hideFreePlan, showJetpackFreePlan );
 
 			plansList = filteredPlans.map( plan => {
+				if ( ! isEnabled( 'upgrades/personal-plan' ) && plan.product_slug === 'personal-bundle' ) {
```

Comment:
that wouldn't work with plans as it would with just products. For plans to work we have to set price configurations on the backend, and that makes them instantly available on `/plans` and `/sites/{siteId}/plans`. 
I could filter it out on those endpoints, and then inject it back on Calypso, but those endpoints are not being used right now by the mobile apps (they will in the future), and there's no access to plans or billing features on the legacy `wp-admin` interface.
Also, is we A/B test multiple variants of the personal plan, all of them have to be configured on the backend as different plans. I'd like to reduce the possible points of failure to just Calypso for now.
I'd rather stick with the backend data for safety.